static/app/components/events/interfaces/spans/spanTree.tsx

```diff
@@ -210,6 +210,8 @@ class SpanTree extends Component<PropType> {
       }
     }, 0);
 
+    const isEmbeddedSpanTree = !!waterfallModel.focusedSpanIds;
```

Comment:
Might be nice to move this to the model I think for clarity, `waterfallModel.isEmbedded()`

Reply:
I wanted to avoid adding even more vars to the waterfallModel since it's already quite cluttered but you're right I should do this